### ISSUE TITLE
linux version?

### ISSUE THREAD

**Issue by Hosseinhashemiir** [peterljq/OpenMMD#5]
it seems an amazing result, but I want to know is there any Linux version?

**Comment by bailiqun** [comment]
Already tested in ubuntu16.04.

**Comment by Hosseinhashemiir** [comment]
Thanks for your attention.
but how you run the .bat file on Ubuntu?
Could you please guide me.

**Comment by bailiqun** [comment]
I focus on how to generate vmd files, so I just run IV step in OpenMMD/VMD 3D Pose Baseline Multi-Objects/applications/. and data in examples. If you focus on project, you need to run whole project step by step~ so there maybe a lot dependcy~

**Comment by peterljq** [comment]
Yep, thank you @bailiqun for answering :D. @erjanmx That will be a good choice if you can focus on the python file component that is most helpful to you. Or you may have to run all the .bat files step by step on Windows. I will program an interface in the later version if possible.

**Comment by Hosseinhashemiir** [comment]
@bailiqun @peterljq Thanks for your attention.
Finally, I was successful to run the code and get a good result.
my issue was I couldn't run the OpenPose code cause for my weak hardware, so finally I used AWS for that.
but there is another question? 
is there any way to convert VMD to another motion files like BVH format?

**Comment by peterljq** [comment]
For myself, I am not quite familiar with other formats like BVH. It will be great if you are willing to develop a relevant toolkit for conversion from vmd to other formats. :D By the way, here is a tool for relevant conversions and read/write in VMD. You may refer to [this tool](https://github.com/ousttrue/pymeshio).

**Comment by KevinLTT** [comment]
@HHIR You can see my project [video2bvh](https://github.com/KevinLTT/video2bvh). It was developed on Linux, and the output format is BVH.
